### ISSUE TITLE
Unsubscribe from events

### ISSUE THREAD

**Issue by sofiter** [danintel/sawtooth-cookiejar#10]
When I tried the code 
`# Unsubscribe from events`
`request = client_event_pb2.ClientEventsUnsubscribeRequest()`

appears to be unreachable. Maybe a second function to separate the while loop and the unsubscribe function would help?

**Comment by danintel** [comment]
Yes it is unreachable because of the while loop.  I put it there in case someone wants to change the code and exit from the while loop.
